SonarQube.Client/Api/DefaultConfiguration.cs

```diff
@@ -54,5 +54,29 @@ public static RequestFactory Configure(RequestFactory requestFactory)
                 .RegisterRequest<IGetTaintVulnerabilitiesRequest, V8_6.GetTaintVulnerabilitiesRequest>("8.6");
             return requestFactory;
         }
+
+        public static UnversionedRequestFactory ConfigureSonarCloud(UnversionedRequestFactory requestFactory)
```

Comment:
Initially I was going to implement a specific `IRequestFactory` class for SonarCloud. It wouldn't have needed to support dynamic registration, and the mapping of requests-to-factory functions could just have been a `readonly` dictionary that class.

In the end I decided it would be useful to have a single file in which we could see what was registered for each service i.e. this one. I'm not sure it was the right decision - time will tell.

That decision meant I needed to add a new type of "unversioned factory". This is actually just a cut-down version of the current `RequestFactory` that doesn't support registering multiple versions for each request.